sktime/tests/test_all_estimators.py

```diff
@@ -662,6 +685,20 @@ def test_has_common_interface(self, estimator_class):
         if hasattr(estimator, "predict_proba"):
             assert hasattr(estimator, "predict")
 
+    def test_no_cross_test_side_effects_part1(self, estimator_instance):
+        """Tests that there are no side effects across tests."""
```

Comment:
Docstrings should start with bare infinitive, so 'Test ...', in the ones below as well. The wording in the PR also seems clearer I think, `Test if estimator instance state is preserved across tests or test instances.` Other than that looks all good to me

Reply:
ok, I updated the docstring.

I did not go with your suggestion, since I think that can be read to imply that "preserving the state" (preserving being a positive verb) is the desired condition.

Reply:
Sure, makes sense